src/arkode/arkode_io.c

```diff
@@ -1743,6 +1749,11 @@ int arkPrintAllStats(void *arkode_mem, FILE *outfile, SUNOutputFormat fmt)
       ark_root_mem = (ARKodeRootMem) ark_mem->root_mem;
       fprintf(outfile, ",Roof fn evals,%ld", ark_root_mem->nge);
     }
+    if (ark_mem->relax_enabled)
+    {
+      retval = arkRelaxPrintAllStats(arkode_mem, outfile, fmt);
+      if (retval != ARK_SUCCESS) return(retval);
+    }
```

Comment:
Instead of duplicating this block in both `case` statements, can we just move it after the end of the `switch` block entirely?